scripts/fusion-swap/fill.ts

```diff
@@ -59,15 +56,27 @@ async function fill(
     takerKeypair.publicKey
   );
 
+  const takerDstAta = await splToken.getAssociatedTokenAddress(
+    orderConfig.dstMint,
+    takerKeypair.publicKey
```

Comment:
It seems reasonable to use the taker variable declared on line 74 here and earlier in the code.